.github/workflows/pypi-cd.yml

```diff
@@ -187,13 +186,13 @@ jobs:
               with:
                   working-directory: ./python
                   target: ${{ matrix.build.TARGET }}
-                  args: --release --strip --out wheels -i  ${{ github.event_name != 'pull_request' && 'python3.8 python3.9 python3.10 python3.11 python3.12' || 'python3.10' }}
+                  args: --release --strip --out wheels -i  ${{ github.event_name != 'pull_request' && 'python3.9 python3.10 python3.11 python3.12' || 'python3.10' }}
```

Comment:
Python 3.8 is unavailable in `brew`. Other means of installation could be tricky.